main/subversion/APKBUILD

```diff
@@ -64,12 +64,10 @@ build() {
 }
 
 check() {
-	cd "$builddir"
```

Comment:
This package is in `main` so we keep these `cd` lines to make back porting easier.

Reply:
Done